### PR TITLE
fix data context url tests

### PR DESCRIPTION
**Pull request by Aylr**

### CODE REVIEW

File: tests/data_context/test_data_context.py [hunk 1]
```diff
@@ -1076,53 +1081,6 @@ def test_existing_local_data_docs_urls_returns_multiple_urls_from_customized_loc
     ])
 
 
-def test_existing_local_data_docs_urls_returns_only_existing_urls_from_customized_local_site(tmp_path_factory):
```
Comment (Aylr) [inline tests/data_context/test_data_context.py:1079]: This behavior was no longer desired.

File: tests/data_context/test_data_context.py [hunk 2]
```diff
@@ -996,13 +996,18 @@ def test_build_batch_kwargs(titanic_multibatch_data_context):
...
+    datasource is not configured, and docs are not built.
+    """
+    empty_directory = str(tmp_path_factory.mktemp("another_empty_project"))
     DataContext.create(empty_directory)
     context = DataContext(os.path.join(empty_directory, DataContext.GE_DIR))
 
     obs = context.get_docs_sites_urls()
-    assert obs == []
+    assert len(obs) == 1
+    assert obs[0].endswith("great_expectations/uncommitted/data_docs/local_site/index.html")
```
Comment (jcampbell) [inline tests/data_context/test_data_context.py:1010]: Does this mean it returns a url even if that site does not in fact exist? (i.e. it's never been built?)

**Review by eugmandel — APPROVED** [review]

File: tests/data_context/test_data_context.py (hunk 2 above)
Comment (Aylr) [reply tests/data_context/test_data_context.py:1010]: It does. Here is my rationale on that behavior change. This was built with local disk stores in mind and doesn't use store apis. Therefore unexpected things might happen if the docs are a remote blob store. If the user has a site configured, the url could be retrieved regardless of build status.

Comment (Aylr) [reply tests/data_context/test_data_context.py:1010]: Put another way, I could imagine this being used programmatically and if we need to augment this to only return **built and existing urls** we could add a parameter, or create another method like `get_existing_data_docs_urls`.

**Pull request merged**